cql/DashboardLibrary.cql

```diff
@@ -162,6 +162,9 @@ define function CytologyInterpretation(conclusions List<System.Concept>):
         when c ~ "ASC-H" then {text: 'ASC-H', rank: 5}
         when c ~ "AGC" then {text: 'AGC', rank: 6}
         when c in "HSIL" or c ~ "SCC" then {text: 'HSIL+', rank: 7}
+        when c ~ "AGC Favor Neoplasia" then {text: 'AGC-FN', rank: 8}
```

Comment:
What about every other result in `RecentRareCytologyResults` (copied below)? This only includes 3 of them. Although only those 3 may be able to enterred via the pap tracker, we may see other results from UMMC and other pilots.

```
define RecentRareCytologyResults:
  Common.LookBack(
    (CervicalCytologyReports) D
      where AnyTrue(
        (CollateConclusionCodes(D,CytologyObservations)) cC return
          cC ~ "Benign Glandular Cells" or
          cC ~ "Benign Endometrial Cells" or
          cC ~ "Endometrial Stromal Cells" or
          cC ~ "Histiocytes" or
          cC ~ "AGC Favor Neoplasia" or
          cC ~ "Endocervical Cells Favor Neoplasia" or
          cC ~ "Atypical Endocervical Cells" or
          cC in "AIS"
      ),
    AbnormalTestLookbackPeriod
  )
```

Reply:
I still don't see the following codes from `RecentRareCytologyResults `:
- `Benign Glandular Cells`
- `Endometrial Stromal Cells`

Is there a reason those were left out of `CytologyInterpretation`?

Reply:
I do not see any other test result codes for cytology that are not covered by this PR.